concourse/pipelines/destroy-microbosh.yml

```diff
@@ -98,6 +98,23 @@ jobs:
           - get: bosh-secrets
           - get: bosh-init-state
           - get: bosh-manifest
+      - task: bosh-cleanup
+        config:
+          platform: linux
+          image: docker:///governmentpaas/bosh-cli
+          inputs:
+            - name: paas-cf
+            - name: bosh-secrets
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/bosh_login.sh bosh-secrets/bosh-secrets.yml
+
+                # Cleanup ensures any orphaned disks are deleted before bosh itself is deleted
```

Comment:
`bosh cleanup` help in the bosh cli is also misleading :):

https://github.com/cloudfoundry/bosh/blob/d2159dca24cacbf2731ecd6b4d0d5545b16c678e/bosh_cli/lib/cli/commands/maintenance.rb#L9